spec/forms/base_form_spec.rb

```diff
@@ -84,4 +84,23 @@
       end
     end
   end
+
+  context 'with non-existing attributes' do
+    let(:user_email) { 'asya@purple-magic.com' }
+    let(:object) { create :user, email: user_email }
+    subject { described_class.new(object) }
```

Comment:
Why do we need `subject` if we are not using construction `it { is_expected.to .. }`

Reply:
Didn't know about `is_expected.to`. WIll try :slightly_smiling_face:

Reply:
@tkarpesh `is_expected.to` is a nice thing :slightly_smiling_face: thank you :1st_place_medal: